server/src/common/error.go

```diff
@@ -40,6 +41,15 @@ func ForbiddenError(err error) *APIError {
 	}
 }
 
+func GoneError(err error) *APIError {
```

Comment:
just as mentioned in server/src/api/context.go, maybe not the right error for the job